core/src/main/res/layout/player_widget.xml

```diff
@@ -13,7 +13,7 @@
         tools:ignore="UselessParent">
 
         <ImageButton
-            android:id="@+id/butPlay"
+            android:id="@+id/playButton"
```

Comment:
> The failed tests are pointing to line no 98, but that line is different than the one these failed tests are showing. Please have a look.

You renamed the button here in the widget xml file, too. Please only rename in the scope of this PR (audioplayer)

Additionally, something went wrong while merging again. Your change now reverts some other PRs (have a look at the [files section](https://github.com/AntennaPod/AntennaPod/pull/4854/files) of the PR)